Apps/WebClient/src/Server/Controllers/NoteController.cs

```diff
@@ -58,19 +58,19 @@ public class NoteController
         /// Posts a patient note json to be inserted into the database.
         /// </summary>
         /// <returns>The http status.</returns>
+        /// <param name="hdid">The user hdid.</param>
         /// <param name="note">The patient note request model.</param>
         /// <response code="200">The note record was saved.</response>
         /// <response code="401">The client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpPost]
-        [Authorize(Policy = UserPolicy.UserOnly)]
-        public IActionResult CreateNote([FromBody] UserNote note)
+        [Route("{hdid}")]
+        [Authorize(Policy = UserProfilePolicy.Write)]
```

Comment:
It would be great to have Create policy here for Create...

Reply:
Same as above. Write corresponds to modifying a record, where the read is only about retrieving it